labs/RivePlayer/src/RivePlayer.xaml

```diff
@@ -0,0 +1,28 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:Rive="using:CommunityToolkit.Labs.WinUI.Rive"
+                    xmlns:controls="using:CommunityToolkit.Labs.WinUI"
+                    xmlns:ui="using:Microsoft.Toolkit.Uwp.UI"
+                    xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+
+    <Style TargetType="Rive:RivePlayer">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Rive:RivePlayer">
+                    <ContentPresenter x:Name="SkiaSurface"
+                                      Padding="{TemplateBinding Padding}"
+                                      HorizontalContentAlignment="Stretch"
+                                      VerticalContentAlignment="Stretch"
+                                      win:AutomationProperties.AccessibilityView="Raw"
+                                      Background="{TemplateBinding Background}"
```

Comment:
Nit: should we provide a default BackgroundBrush? I noticed that at times it took a moment to load the content, so having some sort of indication that something is loading might be helpful? (A ProgressRing could be an option as well)?